2022-02-23 23:44:55,234 INFO    MainThread:8503 [wandb_setup.py:_flush():71] setting env: {}
2022-02-23 23:44:55,234 INFO    MainThread:8503 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '***REDACTED***'}
2022-02-23 23:44:55,234 INFO    MainThread:8503 [wandb_setup.py:_flush():71] setting login settings: {'api_key': '***REDACTED***'}
2022-02-23 23:44:55,234 INFO    MainThread:8503 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-23 23:44:55,234 INFO    MainThread:8503 [wandb_init.py:_log_setup():371] Logging user logs to /root/konst/ml-utils/training/non_personal_text/wandb/run-20220223_234455-3v7y34q9/logs/debug.log
2022-02-23 23:44:55,234 INFO    MainThread:8503 [wandb_init.py:_log_setup():372] Logging internal logs to /root/konst/ml-utils/training/non_personal_text/wandb/run-20220223_234455-3v7y34q9/logs/debug-internal.log
2022-02-23 23:44:55,234 INFO    MainThread:8503 [wandb_init.py:init():404] calling init triggers
2022-02-23 23:44:55,234 INFO    MainThread:8503 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {'data_path': '/root/konst/data/non_personal_text/non_personal_texts.csv', 'checkpoint_path': '/root/konst/model_checkpoints/non_personal_text', 'model_path': '', 'batch_size': 128, 'epochs': 100, 'start_epoch': 0, 'test_size': 0.2, 'lam': 0, 'patience': 12, 'factor': 0.5, 'use_wandb': True, 'lr': 0.001, 'save_at': 1, 'adjust_lr_every': 1, 'num_workers': 0}
2022-02-23 23:44:55,234 INFO    MainThread:8503 [wandb_init.py:init():460] starting backend
2022-02-23 23:44:55,234 INFO    MainThread:8503 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-23 23:44:55,240 INFO    MainThread:8503 [backend.py:ensure_launched():216] starting backend process...
2022-02-23 23:44:55,244 INFO    MainThread:8503 [backend.py:ensure_launched():222] started backend process with pid: 8518
2022-02-23 23:44:55,245 INFO    MainThread:8503 [wandb_init.py:init():469] backend started and connected
2022-02-23 23:44:55,249 INFO    MainThread:8503 [wandb_init.py:init():533] updated telemetry
2022-02-23 23:44:55,259 INFO    MainThread:8503 [wandb_init.py:init():563] communicating current version
2022-02-23 23:44:55,862 INFO    MainThread:8503 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-23 23:44:55,862 INFO    MainThread:8503 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-23 23:44:56,068 INFO    MainThread:8503 [wandb_init.py:init():606] starting run threads in backend
2022-02-23 23:44:56,862 INFO    MainThread:8503 [wandb_run.py:_console_start():1810] atexit reg
2022-02-23 23:44:56,862 INFO    MainThread:8503 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.REDIRECT
2022-02-23 23:44:56,863 INFO    MainThread:8503 [wandb_run.py:_redirect():1689] Redirecting console.
2022-02-23 23:44:56,864 INFO    MainThread:8503 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-23 23:44:56,864 INFO    MainThread:8503 [wandb_init.py:init():633] run started, returning control to user process
2022-02-24 09:51:39,085 INFO    MainThread:8503 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 255
2022-02-24 09:51:39,086 INFO    MainThread:8503 [wandb_run.py:_restore():1752] restore
